src/Features/CSharp/Portable/MakeMethodAsynchronous/CSharpMakeMethodAsynchronousCodeFixProvider.cs

```diff
@@ -81,6 +81,10 @@ protected override bool IsAsyncSupportingFunctionSyntax(SyntaxNode node)
                 {
                     newReturnType = taskType.GenerateTypeSyntax();
                 }
+                else
```

Comment:
💭 There is a similar case a few lines below, which applies to methods that return `Task` before they are make asynchronous. I'm guessing you could create a new unit test which fails unless an `else` case is added there too.